[UberNewsroomBridge] Add more regions & use region name from JSON

VerifiedJoseph opened the pull request:
Adds more regions. 
Uses JSON data for region name and article image
Removes `getRegionName()` and `getImage()`

VerifiedJoseph committed:
[UberNewsroomBridge] Add more regions

VerifiedJoseph committed:
[UberNewsroomBridge] Use image from JSON

VerifiedJoseph committed:
[UberNewsroomBridge] Use region name from JSON

yamanq commented:
I don't know if these are the best values to add. For example, for China, you added en-CN, but wouldn't make more sense to use zh-CN? Similarly, for Morocco, ar-MA.

VerifiedJoseph commented:
> [..] but wouldn't make more sense to use zh-CN? Similarly, for Morocco, ar-MA.

It would, I'll fix it. The script I wrote to fetch region data from [newsroom/v1/data](https://newsroomapi.uber.com/wp-json/newsroom/v1/data) used the locale from each of the region's link value instead of the `default_language` value.

yamanq commented:
Oh, that would be great. It looks like for Morocco that would end up being fr-MA, which also works. I didn't check the other locales.

VerifiedJoseph committed:
[UberNewsroomBridge] Use default language as region locales

Updates region locales to use `default_language` value from regions in https://newsroomapi.uber.com/wp-json/newsroom/v1/data

yamanq commented:
I don't understand why some locales are just language while others include the country code, but they seem to work. Thanks for going through them.
